Allow longer names for panels #892

HyungJon opened the pull request:
Fixes #892

dariusf commented on src/test/java/guitests/PanelRenameTest.java:
```diff
@@ -80,121 +77,32 @@ public void panelRenameTest() {
-        boolean isPresentBeforeEdit = exists("#dummy/dummy_col8_closeButton");
-        PlatformEx.runAndWait(() -> UI.events.triggerEvent(new ShowRenamePanelEvent(8)));
+        press(CREATE_RIGHT_PANEL);
+        boolean isPresentBeforeEdit = exists("#dummy/dummy_col3_closeButton");
```
We most likely need the wait. It was added during the meeting just now.

Also pushKeys was used because press causes some problems with the CI (#1047).

dariusf commented:
Also I think you accidentally committed one of the TestFX screenshots.

HyungJon commented:
Right, fixed both mistakes. As for the fall in coverage, I removed all tests regarding name validation and since it's not very important to test how the names are wrapped I didn't write any new ones. If some other features could be tested I'll add them.

dariusf commented:
Okay, looks good.

dariusf commented:
Merged in 43431e58ba1d55628805df6620a218f8964a3845.
